.dockerignore

```diff
@@ -9,3 +9,9 @@
 !.pylintrc
 !requirements.txt
 !requirements_dev.txt
+
+.dependabot
```

Comment:
The docker ignores whitelist should ignore all files that are not explicitly included in the docker ignore. The following snippet should be included at the top of this file (it is missing). 

```yaml
# Ignore everything
*
```

If you add the above snippet to the top of the `.dockerignore` file, you can remove the below files as they won't be copied as they are excluded from the whitelist.

see https://kevinpollet.dev/posts/how-to-use-your-dockerignore-as-a-whitelist/ for more on docker whitelists.

Reply:
`.dockerignore` is updated. The thing is that currently this file is ignored because of `volume` in `docker-compose`.